gapipy/resources/tour/tour_dossier.py

```diff
@@ -90,3 +90,8 @@ def get_trip_detail(self, label):
         for detail in self.details:
             if detail['detail_type']['label'] == label:
                 return detail['body']
+
+    def get_category(self, label):
```

Comment:
@rafikdraoui can this be changed `get_category_name`, as it is more descriptive for what is being returned.

Reply:
This was actually my thought at first, but I decided to stay consistent with how `get_trip_detail` was named. But I don't need more convincing to change it to a more descriptive name (if I really cared that much about consistency, I guess I would have named it `get_trip_category`!)

Reply:
fair enough 👍